RedHatInsights/compliance-frontend
# fix: Fix not-equal sign in error states

Fixes https://issues.redhat.com/browse/RHICOMPL-3415.

This makes `ErrorCard` component render the Something went wrong message if `POST /graphql` return the error code different from 404. Also, incorrect error message parameter passing is fixed.

## How to test

Run this PR together with Inventory locally. In dev tools, intentionally block the `POST /graphql` request. Go to the Inventory Details page for some system and open Compliance tab. You should see the "Something went wrong" message instead of the only not-equal sign reported in the bug ticket.

## Before the change

![image](https://user-images.githubusercontent.com/31385370/210991857-660126a7-c0fb-4635-abff-3a187b8aded9.png)

## After the change

![image](https://user-images.githubusercontent.com/31385370/210991601-b63f9532-939d-42b6-b4a6-2f82cafaeb3a.png)
